CLDR-15715 Survey Tool hung threads; update cldr-apps-webdriver

btangmu opened the pull request:
-Update/add some dependencies in maven.yml and pom.xml

-Change row prefix from r@ to row_ to match change in Survey Tool

-Revise SurveyDriverLog.java to use java.util.logging

btangmu committed:
CLDR-15707 Survey Tool hung threads; update cldr-apps-webdriver

-Update/add some dependencies in maven.yml and pom.xml

-Change row prefix from r@ to row_ to match change in Survey Tool

-Revise SurveyDriverLog.java to use java.util.logging

srl295 commented on .github/workflows/maven.yml:
```diff
@@ -33,7 +33,7 @@ jobs:
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
           java-version: 11
```
See my change in CLDR - need  "distribution: temurin" too

btangmu replied:
thanks, I've just added it

btangmu committed:
CLDR-15707 Survey Tool hung threads; update cldr-apps-webdriver

-Update/add some dependencies in maven.yml and pom.xml

-Change row prefix from r@ to row_ to match change in Survey Tool

-Revise SurveyDriverLog.java to use java.util.logging

btangmu committed:
CLDR-15715 Survey Tool hung threads; update cldr-apps-webdriver

-Update/add some dependencies in maven.yml and pom.xml

-Change row prefix from r@ to row_ to match change in Survey Tool

-Revise SurveyDriverLog.java to use java.util.logging
